subsys/net/lib/lwm2m/ipso_light_control.c

```diff
@@ -10,9 +10,9 @@
  * Section: "16. IPSO Object: Light Control"
  */
 
-#define SYS_LOG_DOMAIN "ipso_light_control"
-#define SYS_LOG_LEVEL CONFIG_SYS_LOG_LWM2M_LEVEL
-#include <logging/sys_log.h>
+#define LOG_MODULE_NAME ipso_light_control
+#define LOG_LEVEL CONFIG_SYS_LOG_LWM2M_LEVEL
```

Comment:
Do we want to change the Kconfig naming here to NET_LOG_LWM2M_LEVEL to match the rest of the subsys/net/lib options?

Reply:
Hmm, I think I will leave to you to decide. I will remove the lwm2m patches from this PR, you can re-use them if you wish.